.github/labeler.yml

```diff
@@ -4,22 +4,10 @@
 
 Python:
   - 'python/**'
-  - 'notebooks/**'
-  
-librmm:
-  - 'cpp/**'
```

Comment:
I believe this should be maintained, @harrism what label should we use here if any?

Reply:
We should probably create a `cpp` label for this